nodejs/v10-community/health-check/booster.yaml

```diff
@@ -4,10 +4,6 @@ source:
   git:
     url: https://github.com/nodeshift-starters/nodejs-health-check
     ref: master
-metadata:
```

Comment:
Are you sure this change belongs to the same PR?

Reply:
I think @tinakurian was trying something out, this doesn't belong here. :)